src/RouteList.js

```diff
@@ -0,0 +1,54 @@
+'use strict'
+
+const Ioc = require('adonis-fold').Ioc
+const Command = Ioc.use('Adonis/Src/Command')
```

Comment:
Can u please inject `Command` and `Routes` inside the constructor and pass it like the way you did with the `Helpers`